### ISSUE TITLE
Separate out "extras" nbs for calculus

### ISSUE THREAD

**Issue by charlesfrye** [wandb/edu#34]
Retain only the components that are easily autograde-able or could be converted to autograding.

**Comment by charlesfrye** [comment]
Consider dropping:
- backprop/computational graph material
- optimization by hand versus by gradient descent
- second-order properties material at the top
